src/qml/imports/Theme/QfBadge.qml

```diff
@@ -0,0 +1,28 @@
+import QtQuick
+import QtQuick.Controls
+
+Rectangle {
+  property alias badgeText: inText
+
+  anchors {
+    top: parent.top
+    right: parent.right
+    rightMargin: 2
+    topMargin: 2
+  }
+
+  width: 12
+  height: 12
+  radius: width / 2
+
+  border.width: 1.5
+  border.color: "white"
```

Comment:
I think I'd stop forcing white here and instead use a Theme color, I think the theme's main background color would be perfect (it's near-white on light theme).

Reply:
I very much love the dark border line, I don't think we need extra logic :)